docs/rules/assertions.md

```diff
@@ -162,23 +154,29 @@ rules:
       - description
 ```
 
-### `context` example
+### `if` example
 
 The following example asserts that PUT responses with HTTP status 200 or 201 cannot return an `application/pdf`content type.
-Without the `context`, the assertion would evaluate every MediaTypesMap including:
+Without the `if`, the assertion would evaluate every MediaTypeMap including:
 - Responses with all codes, including codes other than 200 or 201
 - Responses for all HTTP methods, including DELETE, GET, POST, and more.
-To restrict the evaluation, use the `context` feature to limit what will be evaluated.
+To restrict the evaluation, use the `if` feature to limit what will be evaluated.
 
 ```yaml
 assert/no-pdf-in-ok-response:
-  context:
-  - type: Operation
-    matchParentKeys: [put]
-  - type: Response
-    matchParentKeys: ['201', '200']
-  subject: MediaTypesMap
-  disallowed: ['application/pdf']
+  if:
```

Comment:
For me it looks like if we have `if` statement we also can have `else`. Maybe `context` suits here more the `if`.

Reply:
maybe `given`

Reply:
I think `given` suits more than `if`.  Same for [where](https://github.com/Redocly/redocly-cli/pull/857#discussion_r979583093). But as for me `where` is better than `given` )))

Reply:
I prefer `where` too.